.travis.yml

```diff
@@ -103,6 +103,11 @@ jobs:
     - touch target/doc/std/option/enum.Option.html
     - touch target/doc/std/primitive.usize.html
     - touch target/doc/std/primitive.char.html
+    - touch target/doc/trait.BinaryKey.html
+    - touch target/doc/trait.BinaryValue.html
+    - touch target/doc/struct.ListIndex.html
```

Comment:
Why are the new fake files necessary? As far as I can see, the relevant links indeed don't work; e.g., the link to `../struct.ListIndex.html` does not work for `SparseListIndex` if it is accessed from the crate root. Same for the link to `BinaryKey` from `KeySetIndex`.